### PR TITLE
Updates the UI of sign up page on elm side.

### PR DESCRIPTION
**Pull request by Shrikant9**
Issue [#429](https://github.com/bigbinary/acehelp/issues/429)

### CODE REVIEW

**Comment by Shrikant9** [comment]
@breezyboa _a

File: app/javascript/Admin/Page/Session/SignUp.elm [hunk 1]
```diff
@@ -138,50 +134,80 @@ signUp model =
...
-        [ div [] [ h2 [] [ text "Sign Up" ] ]
-        , Html.form [ onSubmit SignUp ]
-            [ div []
-                [ label [] [ text "First Name: " ]
-                , input
-                    [ Html.Attributes.value <| Field.value model.firstName
-                    , type_ "text"
-                    , placeholder "First Name"
-                    , onInput FirstNameInput
+    div [ class "pageWrap" ]
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Session/SignUp.elm]: do not use cameCase for css classes. use a hypen

File: app/javascript/Admin/Page/Session/SignUp.elm [hunk 2]
```diff
@@ -138,50 +134,80 @@ signUp model =
...
-                ]
-            , div []
-                [ label [] [ text "Confirm Password: " ]
-                , input
-                    [ Html.Attributes.value <| Field.value model.confirmPassword
-                    , type_ "password"
-                    , placeholder "Confirm Password"
-                    , onInput ConfirmPasswordInput
+                , div [ class "centerForm col-md-12" ]
+                    [ Html.form [ onSubmit SignUp ]
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Session/SignUp.elm]: Remove the form as we do not need it

File: app/javascript/Admin/Page/Session/SignUp.elm [hunk 3]
```diff
@@ -138,50 +134,80 @@ signUp model =
...
+                                , placeholder "Confirm Password"
+                                , onInput ConfirmPasswordInput
+                                , class "form-control"
+                                ]
+                                []
+                            ]
+                        , div [ class "formSection" ]
+                            [ button
+                                [ type_ "submit"
+                                , class "btn btn-primary"
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Session/SignUp.elm]: use type "button" and onClick `SignUp`

File: app/javascript/Admin/Page/Session/SignUp.elm [hunk 4]
```diff
@@ -138,50 +134,80 @@ signUp model =
...
+                            [ span [ class "btn col-md-6" ]
+                                [ Html.a
+                                    [ href "/users/sign_in" ]
+                                    [ span [] [ text "Sign In" ] ]
+                                ]
+                            , span [ class "btn" ]
+                                [ Html.a
+                                    [ href "/users/forgot_password" ]
+                                    [ span [] [ text "Forgot password" ] ]
+                                ]
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Session/SignUp.elm]: Do not use anchor tags to redirect. Redirection should happen through elm

**Comment by Shrikant9** [comment]
@breezyboa _a
Email validation along with others will be resolved as a solution to [#467](https://github.com/bigbinary/acehelp/issues/467) issue.

**Pull request merged**